components/RuleDisputeForm/RuleDisputeForm.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { useState } from 'react';
+
+// function App() {
+//   // Similar to useState but first arg is key to the value in local storage.
+//   const [name, setName] = useLocalStorage('name', 'Bob');
+//
+//   return (
+//     <div>
+//       <input
+//         type="text"
+//         placeholder="Enter your name"
+//         value={name}
+//         onChange={e => setName(e.target.value)}
+//       />
+//     </div>
+//   );
+// }
+
+
+
+const RuleDisputeForm = () => {
+  const [expansion, setExpansion] = useLocalStorage('expansion', 'setExpansion')
+  // const [number, setNumber] = useLocalStorage('number', 'setNumber')
+    return(
+      <section>
+      <form>
+        <label>Expansion Name: </label>
+        <input
+        type='text'
+        name='expansion'
+        value={expansion}
+        onChange={event =>setExpansion(event.target.value)}
+        />
+        <label>Number of players: </label>
+        <input
+        type='text'
+        name='number'
+        value={number}
+        onChange={event =>setNumber(event.target.value)}
+        />
+        <label>Gameplay time: </label>
+        <input
+        type='time'
+        name='time'
+        />
+        <label>Official Rule: </label>
+        <input
+        type='text'
+        name='officialRule'
+        />
+        <label>Interpretation of Rule: </label>
+        <input
+        type='text'
+        name='InterpretationRule'
+        />
+
+        <button>SUBMIT</button>
```

Comment:
Do we want to keep SUBMIT or go with Submit or as we have in the design Start Fight?